llvm/lib/CodeGen/GlobalISel/IRTranslator.cpp

```diff
@@ -2789,20 +2789,35 @@ bool IRTranslator::translateCall(const User &U, MachineIRBuilder &MIRBuilder) {
   if (translateKnownIntrinsic(CI, ID, MIRBuilder))
     return true;
 
+  TargetLowering::IntrinsicInfo Info;
+  // TODO: Add a GlobalISel version of getTgtMemIntrinsic.
+  bool IsTgtMemIntrinsic = TLI->getTgtMemIntrinsic(Info, CI, *MF, ID);
```

Comment:
Nothing is preventing use of the current one?

Reply:
I couldn't find one, tbh. How is it called?
(I kept this todo because it was in the original code. It has been added in https://reviews.llvm.org/D33724.)

Reply:
I don't see what another version would be. This doesn't depend on any DAG nodes. Maybe there are some fields in Info that use native ISD types, but that doesn't justify a separate API

Reply:
(I removed the TODO in https://github.com/llvm/llvm-project/pull/153204)